manager/manager.py

```diff
@@ -598,7 +598,7 @@ def generate_dashboard(self, max_failures=MAX_FAILURES):
                 '<th class="hdr">Repo</th>'
                 '<th class="hdr">PreConvert</th>'
                 '<th class="hdr">Converted</th>'
-                '<th class="hdr">Destination</th>'
+                '<th class="hdr">Destination</th>',
```

Comment:
Totally agree, I strongly dislike the string concat method used -- but in the end it is a style issue. 
 Rewriting for style would have to be another issue.